PittAPI/shuttle.py

```diff
@@ -18,32 +18,33 @@
 '''
 
 import requests
+from typing import List, Dict
 
 sess = requests.session()
 
 
-def get_map_vehicle_points(api_key="8882812681"):
+def get_map_vehicle_points(api_key: str = "8882812681") -> Dict[str,Any]:
     """Return the map location for all active vehicles."""
     payload = {"ApiKey": api_key}
     response = sess.get("http://www.pittshuttle.com/Services/JSONPRelay.svc/GetMapVehiclePoints", params=payload)
     return response.json()
 
 
-def get_route_stop_arrivals(api_key="8882812681", times_per_stop=1):
+def get_route_stop_arrivals(api_key: str = "8882812681", times_per_stop: int = 1) -> Dict[str,Any]:
     """Return stop arrival times for all vehicles."""
     payload = {"ApiKey": api_key, "TimesPerStopString": times_per_stop}
     response = sess.get("http://www.pittshuttle.com/Services/JSONPRelay.svc/GetRouteStopArrivals", params=payload)
     return response.json()
 
 
-def get_vehicle_route_stop_estimates(vehicle_id, quantity=2):
+def get_vehicle_route_stop_estimates(vehicle_id: str, quantity: int = 2) -> Dict[str,Any]:
     """Return {quantity} stop estimates for all active vehicles."""
-    payload = {"vehicleIdStrings": vehicle_id, "quantity": quantity}
+    payload = {"vehicleIdStrings": vehicle_id, "quantity": str(quantity)}
     response = sess.get("http://www.pittshuttle.com/Services/JSONPRelay.svc/GetVehicleRouteStopEstimates", params=payload)
     return response.json()
 
 
-def get_routes(api_key="8882812681"):
+def get_routes(api_key: str ="8882812681") -> Dict[str,Any]:
```

Comment:
Again, don't forget to import `Any`, it's giving me a `NameError` for this is the test log.